2020-10-21T09:24:05.9672293+07:00  [INF] Now listening on: "http://localhost:5123" (d826f4b8)
2020-10-21T09:24:05.9752201+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-10-21T09:24:05.9755440+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-10-21T09:24:05.9755874+07:00  [INF] Content root path: "D:\01.SOURCE\00.DEMO\DEM\DEM.Web" (b5d60022)
2020-10-21T09:40:35.3426901+07:00  [INF] Now listening on: "http://localhost:5123" (d826f4b8)
2020-10-21T09:40:35.3488649+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-10-21T09:40:35.3489810+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-10-21T09:40:35.3490218+07:00  [INF] Content root path: "D:\01.SOURCE\00.DEMO\DEM\DEM.Web" (b5d60022)
2020-10-21T10:46:16.8717301+07:00  [INF] Now listening on: "http://localhost:5123" (d826f4b8)
2020-10-21T10:46:16.8780353+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-10-21T10:46:16.8782075+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-10-21T10:46:16.8782443+07:00  [INF] Content root path: "D:\01.SOURCE\00.DEMO\DEM\DEM.Web" (b5d60022)
2020-10-21T10:47:47.5502653+07:00  [INF] Now listening on: "http://localhost:5123" (d826f4b8)
2020-10-21T10:47:47.5610056+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-10-21T10:47:47.5633173+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-10-21T10:47:47.5655455+07:00  [INF] Content root path: "D:\01.SOURCE\00.DEMO\DEM\DEM.Web" (b5d60022)
2020-10-21T10:50:22.0134075+07:00 0HM3LEH42H0D9:0000000E [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.NotImplementedException: The method or operation is not implemented.
   at DEM.App.IntendedService.LoadData(String rootCategory) in D:\01.SOURCE\00.DEMO\DEM\DEM.Business\Implements\IntendedService.cs:line 45
   at DEM.Web.Controllers.IntendedController.LoadData(DataSourceRequest request) in D:\01.SOURCE\00.DEMO\DEM\DEM.Web\Controllers\IntendedController.cs:line 55
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-10-21T11:10:09.7023179+07:00 0HM3LEH42H0DC:00000004 [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.NotImplementedException: The method or operation is not implemented.
   at DEM.App.IntendedService.LoadData(String rootCategory) in D:\01.SOURCE\00.DEMO\DEM\DEM.Business\Implements\IntendedService.cs:line 45
   at DEM.Web.Controllers.IntendedController.LoadData(DataSourceRequest request) in D:\01.SOURCE\00.DEMO\DEM\DEM.Web\Controllers\IntendedController.cs:line 55
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-10-21T11:10:26.4301424+07:00 0HM3LEH42H0DE:00000012 [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.NotImplementedException: The method or operation is not implemented.
   at DEM.App.IntendedService.LoadData(String rootCategory) in D:\01.SOURCE\00.DEMO\DEM\DEM.Business\Implements\IntendedService.cs:line 45
   at DEM.Web.Controllers.IntendedController.LoadData(DataSourceRequest request) in D:\01.SOURCE\00.DEMO\DEM\DEM.Web\Controllers\IntendedController.cs:line 55
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-10-21T11:15:21.9259430+07:00  [INF] Now listening on: "http://localhost:5123" (d826f4b8)
2020-10-21T11:15:21.9385013+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-10-21T11:15:21.9427883+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-10-21T11:15:21.9467675+07:00  [INF] Content root path: "D:\01.SOURCE\00.DEMO\DEM\DEM.Web" (b5d60022)
2020-10-21T11:15:52.1667026+07:00 0HM3LEVRVL6K9:00000010 [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.NotImplementedException: The method or operation is not implemented.
   at DEM.App.IntendedService.LoadData(String rootCategory) in D:\01.SOURCE\00.DEMO\DEM\DEM.Business\Implements\IntendedService.cs:line 45
   at DEM.Web.Controllers.IntendedController.LoadData(DataSourceRequest request) in D:\01.SOURCE\00.DEMO\DEM\DEM.Web\Controllers\IntendedController.cs:line 55
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-10-21T11:22:17.0716619+07:00  [INF] Now listening on: "http://localhost:5123" (d826f4b8)
2020-10-21T11:22:17.0799393+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-10-21T11:22:17.0820445+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-10-21T11:22:17.0837562+07:00  [INF] Content root path: "D:\01.SOURCE\00.DEMO\DEM\DEM.Web" (b5d60022)
2020-10-21T11:23:19.4769730+07:00 0HM3LF3TK7PGR:00000010 [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.NotImplementedException: The method or operation is not implemented.
   at DEM.App.IntendedService.LoadData(String rootCategory) in D:\01.SOURCE\00.DEMO\DEM\DEM.Business\Implements\IntendedService.cs:line 45
   at DEM.Web.Controllers.IntendedController.LoadData(DataSourceRequest request) in D:\01.SOURCE\00.DEMO\DEM\DEM.Web\Controllers\IntendedController.cs:line 56
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-10-21T11:24:11.2602689+07:00 0HM3LF3TK7PGN:00000019 [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.NotImplementedException: The method or operation is not implemented.
   at DEM.App.IntendedService.LoadData(String rootCategory) in D:\01.SOURCE\00.DEMO\DEM\DEM.Business\Implements\IntendedService.cs:line 45
   at DEM.Web.Controllers.IntendedController.LoadData(DataSourceRequest request) in D:\01.SOURCE\00.DEMO\DEM\DEM.Web\Controllers\IntendedController.cs:line 56
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-10-21T11:48:25.7799225+07:00  [INF] Now listening on: "http://localhost:5123" (d826f4b8)
2020-10-21T11:48:25.7928672+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-10-21T11:48:25.7966040+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-10-21T11:48:25.7986689+07:00  [INF] Content root path: "D:\01.SOURCE\00.DEMO\DEM\DEM.Web" (b5d60022)
2020-10-21T11:51:04.7589975+07:00 0HM3LFI84353U:00000005 [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.NotImplementedException: The method or operation is not implemented.
   at DEM.App.IntendedService.LoadData(String rootCategory) in D:\01.SOURCE\00.DEMO\DEM\DEM.Business\Implements\IntendedService.cs:line 45
   at DEM.Web.Controllers.IntendedController.LoadData(DataSourceRequest request) in D:\01.SOURCE\00.DEMO\DEM\DEM.Web\Controllers\IntendedController.cs:line 56
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-10-21T11:52:59.5856438+07:00 0HM3LFI843540:00000022 [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.NotImplementedException: The method or operation is not implemented.
   at DEM.App.IntendedService.LoadData(String rootCategory) in D:\01.SOURCE\00.DEMO\DEM\DEM.Business\Implements\IntendedService.cs:line 45
   at DEM.Web.Controllers.IntendedController.LoadData(DataSourceRequest request) in D:\01.SOURCE\00.DEMO\DEM\DEM.Web\Controllers\IntendedController.cs:line 56
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-10-21T11:58:35.0354186+07:00 0HM3LFI843545:0000000D [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.NotImplementedException: The method or operation is not implemented.
   at DEM.App.IntendedService.LoadData(String rootCategory) in D:\01.SOURCE\00.DEMO\DEM\DEM.Business\Implements\IntendedService.cs:line 45
   at DEM.Web.Controllers.IntendedController.LoadData(DataSourceRequest request) in D:\01.SOURCE\00.DEMO\DEM\DEM.Web\Controllers\IntendedController.cs:line 56
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-10-21T11:58:35.0474075+07:00 0HM3LFI843549:00000009 [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.NotImplementedException: The method or operation is not implemented.
   at DEM.App.IntendedService.LoadData(String rootCategory) in D:\01.SOURCE\00.DEMO\DEM\DEM.Business\Implements\IntendedService.cs:line 45
   at DEM.Web.Controllers.IntendedController.LoadData(DataSourceRequest request) in D:\01.SOURCE\00.DEMO\DEM\DEM.Web\Controllers\IntendedController.cs:line 56
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-10-21T13:32:23.8174402+07:00  [INF] Now listening on: "http://localhost:5123" (d826f4b8)
2020-10-21T13:32:23.8405801+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-10-21T13:32:23.8437561+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-10-21T13:32:23.8469448+07:00  [INF] Content root path: "D:\01.SOURCE\00.DEMO\DEM\DEM.Web" (b5d60022)
2020-10-21T13:33:37.9975306+07:00 0HM3LHCE1AO47:00000004 [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.NotImplementedException: The method or operation is not implemented.
   at DEM.App.IntendedService.LoadData(String rootCategory) in D:\01.SOURCE\00.DEMO\DEM\DEM.Business\Implements\IntendedService.cs:line 45
   at DEM.Web.Controllers.IntendedController.LoadData(DataSourceRequest request) in D:\01.SOURCE\00.DEMO\DEM\DEM.Web\Controllers\IntendedController.cs:line 56
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-10-21T13:33:37.9975305+07:00 0HM3LHCE1AO46:00000006 [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.NotImplementedException: The method or operation is not implemented.
   at DEM.App.IntendedService.LoadData(String rootCategory) in D:\01.SOURCE\00.DEMO\DEM\DEM.Business\Implements\IntendedService.cs:line 45
   at DEM.Web.Controllers.IntendedController.LoadData(DataSourceRequest request) in D:\01.SOURCE\00.DEMO\DEM\DEM.Web\Controllers\IntendedController.cs:line 56
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-10-21T13:33:46.2558452+07:00 0HM3LHCE1AO47:00000005 [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.NotImplementedException: The method or operation is not implemented.
   at DEM.App.IntendedService.LoadData(String rootCategory) in D:\01.SOURCE\00.DEMO\DEM\DEM.Business\Implements\IntendedService.cs:line 45
   at DEM.Web.Controllers.IntendedController.LoadData(DataSourceRequest request) in D:\01.SOURCE\00.DEMO\DEM\DEM.Web\Controllers\IntendedController.cs:line 56
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-10-21T13:33:46.2565748+07:00 0HM3LHCE1AO46:00000007 [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.NotImplementedException: The method or operation is not implemented.
   at DEM.App.IntendedService.LoadData(String rootCategory) in D:\01.SOURCE\00.DEMO\DEM\DEM.Business\Implements\IntendedService.cs:line 45
   at DEM.Web.Controllers.IntendedController.LoadData(DataSourceRequest request) in D:\01.SOURCE\00.DEMO\DEM\DEM.Web\Controllers\IntendedController.cs:line 56
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-10-21T14:18:32.7757887+07:00 0HM3LHCE1AO46:00000008 [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.NotImplementedException: The method or operation is not implemented.
   at DEM.App.IntendedService.LoadData(String rootCategory) in D:\01.SOURCE\00.DEMO\DEM\DEM.Business\Implements\IntendedService.cs:line 45
   at DEM.Web.Controllers.IntendedController.LoadData(DataSourceRequest request) in D:\01.SOURCE\00.DEMO\DEM\DEM.Web\Controllers\IntendedController.cs:line 56
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-10-21T14:18:33.7452754+07:00 0HM3LHCE1AO46:00000009 [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.NotImplementedException: The method or operation is not implemented.
   at DEM.App.IntendedService.LoadData(String rootCategory) in D:\01.SOURCE\00.DEMO\DEM\DEM.Business\Implements\IntendedService.cs:line 45
   at DEM.Web.Controllers.IntendedController.LoadData(DataSourceRequest request) in D:\01.SOURCE\00.DEMO\DEM\DEM.Web\Controllers\IntendedController.cs:line 56
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-10-21T16:45:40.0967375+07:00  [INF] Now listening on: "http://localhost:5123" (d826f4b8)
2020-10-21T16:45:40.1027465+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-10-21T16:45:40.1028892+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-10-21T16:45:40.1029400+07:00  [INF] Content root path: "D:\01.SOURCE\00.DEMO\DEM\DEM.Web" (b5d60022)
2020-10-21T16:46:53.2288026+07:00  [INF] Now listening on: "http://localhost:5123" (d826f4b8)
2020-10-21T16:46:53.2332145+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-10-21T16:46:53.2333666+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-10-21T16:46:53.2334403+07:00  [INF] Content root path: "D:\01.SOURCE\00.DEMO\DEM\DEM.Web" (b5d60022)
2020-10-21T16:56:21.0762584+07:00  [INF] Now listening on: "http://localhost:5123" (d826f4b8)
2020-10-21T16:56:21.0864063+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-10-21T16:56:21.0866578+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-10-21T16:56:21.0867363+07:00  [INF] Content root path: "D:\01.SOURCE\00.DEMO\DEM\DEM.Web" (b5d60022)
2020-10-21T16:59:38.2983489+07:00  [INF] Now listening on: "http://localhost:5123" (d826f4b8)
2020-10-21T16:59:38.3082440+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-10-21T16:59:38.3084485+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-10-21T16:59:38.3085064+07:00  [INF] Content root path: "D:\01.SOURCE\00.DEMO\DEM\DEM.Web" (b5d60022)
2020-10-21T16:59:51.0916288+07:00  [INF] Now listening on: "http://localhost:5123" (d826f4b8)
2020-10-21T16:59:51.1018025+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-10-21T16:59:51.1058434+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-10-21T16:59:51.1079111+07:00  [INF] Content root path: "D:\01.SOURCE\00.DEMO\DEM\DEM.Web" (b5d60022)
2020-10-21T17:07:43.6914667+07:00  [INF] Now listening on: "http://localhost:5123" (d826f4b8)
2020-10-21T17:07:43.7017834+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-10-21T17:07:43.7020786+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-10-21T17:07:43.7021330+07:00  [INF] Content root path: "D:\01.SOURCE\00.DEMO\DEM\DEM.Web" (b5d60022)
2020-10-21T17:38:32.2670918+07:00  [INF] Now listening on: "http://localhost:5123" (d826f4b8)
2020-10-21T17:38:32.2729711+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-10-21T17:38:32.2731279+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-10-21T17:38:32.2731714+07:00  [INF] Content root path: "D:\01.SOURCE\00.DEMO\DEM\DEM.Web" (b5d60022)
2020-10-21T17:40:09.3139324+07:00  [INF] Now listening on: "http://localhost:5123" (d826f4b8)
2020-10-21T17:40:09.3250613+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-10-21T17:40:09.3255545+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-10-21T17:40:09.3256569+07:00  [INF] Content root path: "D:\01.SOURCE\00.DEMO\DEM\DEM.Web" (b5d60022)
2020-10-21T21:09:33.2581613+07:00  [INF] Now listening on: "http://localhost:5123" (d826f4b8)
2020-10-21T21:09:33.2630219+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-10-21T21:09:33.2631077+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-10-21T21:09:33.2631362+07:00  [INF] Content root path: "D:\01.TOAN\05.PROJECT\05.FAMILY\DEM\DEM.Web" (b5d60022)
2020-10-21T21:12:35.7129123+07:00  [INF] Now listening on: "http://localhost:5123" (d826f4b8)
2020-10-21T21:12:35.7190442+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-10-21T21:12:35.7202972+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-10-21T21:12:35.7214070+07:00  [INF] Content root path: "D:\01.TOAN\05.PROJECT\05.FAMILY\DEM\DEM.Web" (b5d60022)
2020-10-21T21:15:40.4852452+07:00  [INF] Now listening on: "http://localhost:5123" (d826f4b8)
2020-10-21T21:15:40.4895844+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-10-21T21:15:40.4896428+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-10-21T21:15:40.4896660+07:00  [INF] Content root path: "D:\01.TOAN\05.PROJECT\05.FAMILY\DEM\DEM.Web" (b5d60022)
2020-10-21T21:16:50.3677988+07:00  [INF] Now listening on: "http://localhost:5123" (d826f4b8)
2020-10-21T21:16:50.3721737+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-10-21T21:16:50.3722436+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-10-21T21:16:50.3722654+07:00  [INF] Content root path: "D:\01.TOAN\05.PROJECT\05.FAMILY\DEM\DEM.Web" (b5d60022)
2020-10-21T21:18:12.2857053+07:00 0HM3LPFQDVQEB:0000001D [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.NullReferenceException: Object reference not set to an instance of an object.
   at DEM.App.CategoryConvertMapping.Convert(Category source, CategoryDto destination, ResolutionContext context) in D:\01.TOAN\05.PROJECT\05.FAMILY\DEM\DEM.Business\Maps\Convert\CategoryConvertMapping.cs:line 51
   at AutoMapper.Mapper.MapCore[TSource,TDestination](TSource source, TDestination destination, ResolutionContext context, Type sourceType, Type destinationType, IMemberMap memberMap)
   at AutoMapper.Mapper.Map[TSource,TDestination](TSource source, TDestination destination)
   at AutoMapper.Mapper.Map[TSource,TDestination](TSource source)
   at DEM.App.CategoryService.FindId(Guid id) in D:\01.TOAN\05.PROJECT\05.FAMILY\DEM\DEM.Business\Implements\CategoryService.cs:line 58
   at DEM.Web.Controllers.CategoryController.EditForm(Guid categoryId) in D:\01.TOAN\05.PROJECT\05.FAMILY\DEM\DEM.Web\Controllers\CategoryController.cs:line 35
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeNextActionFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-10-21T21:54:50.3783148+07:00  [INF] Now listening on: "http://localhost:5123" (d826f4b8)
2020-10-21T21:54:50.3827429+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-10-21T21:54:50.3828076+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-10-21T21:54:50.3828233+07:00  [INF] Content root path: "D:\01.TOAN\05.PROJECT\05.FAMILY\DEM\DEM.Web" (b5d60022)
2020-10-21T21:54:59.5745677+07:00 0HM3LQ51RSL0A:00000006 [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.NullReferenceException: Object reference not set to an instance of an object.
   at AspNetCore.Views_Intended__edit.ExecuteAsync() in D:\01.TOAN\05.PROJECT\05.FAMILY\DEM\DEM.Web\Views\Intended\_edit.cshtml:line 51
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.RenderPageCoreAsync(IRazorPage page, ViewContext context)
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.RenderPageAsync(IRazorPage page, ViewContext context, Boolean invokeViewStarts)
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.RenderAsync(ViewContext context)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.ExecuteAsync(ViewContext viewContext, String contentType, Nullable`1 statusCode)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.ExecuteAsync(ViewContext viewContext, String contentType, Nullable`1 statusCode)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.ExecuteAsync(ActionContext actionContext, IView view, ViewDataDictionary viewData, ITempDataDictionary tempData, String contentType, Nullable`1 statusCode)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.PartialViewResultExecutor.ExecuteAsync(ActionContext context, PartialViewResult result)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResultFilterAsync>g__Awaited|29_0[TFilter,TFilterAsync](ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResultExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.ResultNext[TFilter,TFilterAsync](State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeResultFilters()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-10-21T21:55:23.2837545+07:00 0HM3LQ51RSL0B:00000003 [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.NullReferenceException: Object reference not set to an instance of an object.
   at AspNetCore.Views_Intended__edit.ExecuteAsync() in D:\01.TOAN\05.PROJECT\05.FAMILY\DEM\DEM.Web\Views\Intended\_edit.cshtml:line 51
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.RenderPageCoreAsync(IRazorPage page, ViewContext context)
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.RenderPageAsync(IRazorPage page, ViewContext context, Boolean invokeViewStarts)
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.RenderAsync(ViewContext context)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.ExecuteAsync(ViewContext viewContext, String contentType, Nullable`1 statusCode)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.ExecuteAsync(ViewContext viewContext, String contentType, Nullable`1 statusCode)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.ExecuteAsync(ActionContext actionContext, IView view, ViewDataDictionary viewData, ITempDataDictionary tempData, String contentType, Nullable`1 statusCode)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.PartialViewResultExecutor.ExecuteAsync(ActionContext context, PartialViewResult result)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResultFilterAsync>g__Awaited|29_0[TFilter,TFilterAsync](ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResultExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.ResultNext[TFilter,TFilterAsync](State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeResultFilters()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|24_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-10-21T21:55:53.0561489+07:00  [INF] Now listening on: "http://localhost:5123" (d826f4b8)
2020-10-21T21:55:53.0606711+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-10-21T21:55:53.0607304+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-10-21T21:55:53.0607524+07:00  [INF] Content root path: "D:\01.TOAN\05.PROJECT\05.FAMILY\DEM\DEM.Web" (b5d60022)
2020-10-21T22:11:33.1516785+07:00  [INF] Now listening on: "http://localhost:5123" (d826f4b8)
2020-10-21T22:11:33.1569026+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-10-21T22:11:33.1569979+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-10-21T22:11:33.1570252+07:00  [INF] Content root path: "D:\01.TOAN\05.PROJECT\05.FAMILY\DEM\DEM.Web" (b5d60022)
2020-10-21T22:12:52.3868362+07:00  [INF] Now listening on: "http://localhost:5123" (d826f4b8)
2020-10-21T22:12:52.3913233+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-10-21T22:12:52.3913910+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-10-21T22:12:52.3914160+07:00  [INF] Content root path: "D:\01.TOAN\05.PROJECT\05.FAMILY\DEM\DEM.Web" (b5d60022)
2020-10-21T22:13:50.1071038+07:00  [INF] Now listening on: "http://localhost:5123" (d826f4b8)
2020-10-21T22:13:50.1117489+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-10-21T22:13:50.1118423+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-10-21T22:13:50.1118597+07:00  [INF] Content root path: "D:\01.TOAN\05.PROJECT\05.FAMILY\DEM\DEM.Web" (b5d60022)
2020-10-21T22:15:17.2699423+07:00  [INF] Now listening on: "http://localhost:5123" (d826f4b8)
2020-10-21T22:15:17.2762128+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-10-21T22:15:17.2777723+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-10-21T22:15:17.2794798+07:00  [INF] Content root path: "D:\01.TOAN\05.PROJECT\05.FAMILY\DEM\DEM.Web" (b5d60022)
2020-10-21T22:18:00.3671008+07:00  [INF] Now listening on: "http://localhost:5123" (d826f4b8)
2020-10-21T22:18:00.3715556+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-10-21T22:18:00.3716181+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-10-21T22:18:00.3716343+07:00  [INF] Content root path: "D:\01.TOAN\05.PROJECT\05.FAMILY\DEM\DEM.Web" (b5d60022)
2020-10-21T22:20:46.8738455+07:00  [INF] Now listening on: "http://localhost:5123" (d826f4b8)
2020-10-21T22:20:46.8800266+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-10-21T22:20:46.8813118+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-10-21T22:20:46.8825774+07:00  [INF] Content root path: "D:\01.TOAN\05.PROJECT\05.FAMILY\DEM\DEM.Web" (b5d60022)
2020-10-21T22:22:21.5988741+07:00  [INF] Now listening on: "http://localhost:5123" (d826f4b8)
2020-10-21T22:22:21.6032840+07:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-10-21T22:22:21.6033426+07:00  [INF] Hosting environment: "Development" (c3307c92)
2020-10-21T22:22:21.6033617+07:00  [INF] Content root path: "D:\01.TOAN\05.PROJECT\05.FAMILY\DEM\DEM.Web" (b5d60022)
